content/guides/component-testing/quickstart-angular.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Quickstart: Angular'
+---
+
+<CtBetaAlert></CtBetaAlert>
+
+To follow along with this guide, you'll need an Angular CLI application.
```

Comment:
This feels like it's worth wrapping in a banner since it's critical, but not directly related to the below content. Should help if it's more noticeable. 

```suggestion

:::caution

To follow along with this guide, you'll need an Angular CLI application.

:::
```

Reply:
Don't think this is needed as the next paragraph talks about creating the app that will be needed for the rest of the guide.